roles/elastic-stack/ansible-kibana/defaults/main.yml

```diff
@@ -9,6 +9,14 @@ elastic_stack_version: 7.5.1
 wazuh_version: 3.11.0
 wazuh_app_url: https://packages.wazuh.com/wazuhapp/wazuhapp
 
+# API credentials
+api_auth:
```

Comment:
I would go with `wazuh_api_credentials` or `wazuh_api_auth` and remove the `api_` prefix for `id` `url`...

What are your thoughts about it?

Reply:
Well spotted. Fixed in latest commit.